### DISCUSSION TITLE
Welcome to lvim Discussions!

### DISCUSSION THREAD

**Discussion by lvim-tech** [lvim-tech/lvim#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrjbj** [comment]
I'm coming from emacs and evaluating this distro in comparison to other neovim distros like LunaVim and LazyVim.   LazyVim looks best out of box cosmetically, but isn't geared toward developers.  The "message" toast-like component is annoying but the standard config of telescope and the standard bindings for find-file are nice.  lvm-tech has the best leader-key bindings (with the semi-colons) but the out of box telescope config is too small a window and no preview.  Also, the out of box color schemes are pretty bad, if you don't mind me saying so.  Of course, I understand all of this is configurable, and this is an open source project, so apologies for sounding ungrateful or nit-picky here.  It's geared much more toward developers and really shines.  I'm coming from emacs and have having a heck of a time trying to figure out how to customize via lua and the help system is bonkers comparatively speaking.  Perhaps I just need to put in the time to figure out how to find stuff.  The telescope like mapping of keybindings is nice as well.  I think you've done really great work here and wanted to thank you.  I'll definitely be following along and perhaps better able to contribute as I learn more about how to customize things. For now, though, I'll continue to use lazy vim as its easier to add LSP and dev tooling to that than to change the color scheme and telescope settings and other stuff that I have no clue how to alter  here yet.  Either way, great job and thank you!
